dashboard/test/controllers/concerns/azure_text_to_speech_test.rb

```diff
@@ -3,6 +3,7 @@
 
 class AzureTextToSpeechTest < ActionController::TestCase
   setup do
+    skip 'tests have been flaky'
```

Comment:
is this line alone enough to skip all the test cases?

Reply:
Turns out, it works.  Thanks for double-checking.